pkg/cloud/services/ssm/zz_generated.ssm-fetch.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ssm
+
+const ssmFetchScript = `# Copyright 2020 The Kubernetes Authors.
```

Comment:
Are you generating this so you can have the python script as a separate file? Is there any reason we can't just inline the python script in the go code without going through the generation part?